src/components/OrderDetails.js

```diff
@@ -1,14 +1,62 @@
 import { Component, PropTypes, DOM as dom, createFactory } from 'react';
+import { connect } from 'react-redux';
+import actions from '../action_creators';
+import _ from 'underscore';
+
+function mapStateToProps(state) {
+  return {
+    orders: state.orders
```

Comment:
So I'm not sure it's best to have a specific OrderDetails know about / have access to all orders. Ideally, we would just pull the data for the particular order (status, entries, etc.) and map those to props. But I don't think we can access the id param from state right..? That's why I was looking into redux-router. This is probably fine for now though..